code/__DEFINES/_globals.dm

```diff
@@ -53,6 +53,9 @@
 /// Create an untyped null global
 #define GLOBAL_VAR(X) GLOBAL_RAW(/##X); GLOBAL_UNMANAGED(X)
 
+/// Create an untyped null global
+#define GLOBAL_VAR_TYPED(X, Typepath) GLOBAL_RAW(Typepath/##X); GLOBAL_UNMANAGED(X)
```

Comment:
This is just the same as GLOBAL_DATUM, albeit with a better name and an incorrect comment